Automattic/wp-calypso
# Do not show bottom image on coming-soon modals.

## Proposed Changes

* Do not show VideoMaker images on coming-soon video upload modal.
* Update intent images to match figma.

<img width="1718" alt="Screenshot 2023-09-06 at 3 53 43 PM" src="https://github.com/Automattic/wp-calypso/assets/500744/b522829c-228b-4e93-9e79-5e2b2e64b05a">


## Testing Instructions



* visit the onboarding flow and open the video upload coming soom modal. You see no image at the bottom.
* You see images at the bottom for the rest.

## Pre-merge Checklist
